docs/en/operations/system-tables/histogram_metrics.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: /en/operations/system-tables/metrics
```

Comment:
Guessing this should be `slug: /en/operations/system-tables/histogram_metrics`, but I have no idea what this is for.

Reply:
fixed